libghdl_pkg.py.in

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
+
+from os import mkdir
+from os.path import join
+from shutil import copy, copytree
+
+
+libghdl_filename = 'libghdl-' + "@libghdl_version@" + '@SOEXT@'
+dstdir = 'build-libghdl'
+
+# Copy python sources
+copytree(
+    join('@srcdir@/src/vhdl/python/libghdl'),
+    join(dstdir),
+)
+
+# Create config file
+with open(join(dstdir, 'config.py'), 'w') as f:
+    f.write('libghdl_filename="{}"\n'.format(libghdl_filename))
+
+# Copy shared library and VHDL libraries
+copy(libghdl_filename, dstdir)
+copy('setup.py', dstdir)
+
+# Copy VHDL libraries
+def ignore_function(_, lst):
+    return [item for item in lst if item.endswith('.o')]
+copytree(
+    join('lib', 'ghdl'),
+    join(dstdir, 'ghdl'),
+    ignore=ignore_function
+)
```

Comment:
Why not doing that by the makefile ?

Reply:
Actually, `libghdl.py` can be completely integrated in the makefile. But I have not implemented it yet. I did this changes half an hour ago, and I just wanted to show it to you in order to see how you feel about changing the procedure. Also, to hear your thoughts in case I'm doing anything wrong.

If you are ok with removing `class GHDLBuild` from `setup.py`, I will implement it in the makefile, instead of using a separate python script.

Reply:
Yes, I think it would be more readable.